modules/administration-guide/partials/proc_switch_che-to-next-all-namespaces-olm-channel.adoc

```diff
@@ -0,0 +1,89 @@
+[id="switch_che-to-next-all-namespaces-olm-channel_{context}"]
+= Switch {prod} Che to next all namespace OLM channel
+
+{prod} instance installed using OLM can be switched to use "next-all-namespaces" OLM channel for testing new features or bug fixes purpose.
+
+> Alert: {prod} installed from "stable" OLM channel before migration to "next-all-namespace" should be migrated to {devworkspace} engine and this process is not backward compatible. 
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* An instance of {prod-short} running in OpenShift 4 cluster.
+
+.Procedure
+
+Make user migration {link} if your previous OLM channel was "stable" or "next".
```

Comment:
There is inconsistency.
You mentioned `next` here but `tech-preview-stable-all-namespaces` and `next-all-namespaces` below

Reply:
Sorry, I didn't get you concern. We can make migration:
-  from  tech-preview-stable-all-namespaces to next-all-namespaces channel. 
- from stable to next-all-namespaces channel.
- from next to  next-all-namespaces channel.

These are valid cases. For migration from  'next' and 'stable' to next-all-namespaces it's required  user id and user profile migration.